apostrophecms/a3-demo
# Pro 2277 locale selector to allow switching between english and french content

[PRO-2277](https://linear.app/apostrophecms/issue/PRO-2277/as-a-demo-user-i-can-toggle-to-view-content-in-french-or-english)

## Purpose
Config two locales in i18n module, adds a locale selector in global layout if more than two locales.

## How to test
Check that the local selector shows with the two locales, click to switch from one to another.
